src/Dfe.PlanTech.Application/Questionnaire/Queries/GetSectionQuery.cs

```diff
@@ -30,6 +31,12 @@ public class GetSectionQuery : ContentRetriever, IGetSectionQuery
                 }
         };
 
-        return (await repository.GetEntities<Section>(options, cancellationToken)).FirstOrDefault();
+        try
+        {
+            return (await repository.GetEntities<Section>(options, cancellationToken)).FirstOrDefault();
+        } catch (Exception ex)
```

Comment:
There will be 3 main exceptions thrown:
- `ContentfulRateLimitException`
- `GatewayTimeoutException`
- `ContentfulException`

Ideally we should handle these separately, and not just catch everything + throw `ContentfulDataUnavailableException`. Especially since it might make diagnosing problems harder for us.

That said; that's not gonna happen in 1 week so I wouldn't worry about it! Just making note for future reference